pkg/virtctl/vmexport/vmexport.go

```diff
@@ -486,10 +491,27 @@ func DeleteVirtualMachineExport(client kubecli.KubevirtClient, vmeInfo *VMExport
 
 // DownloadVirtualMachineExport handles the process of downloading the requested volume from a VirtualMachineExport object
 func DownloadVirtualMachineExport(client kubecli.KubevirtClient, vmeInfo *VMExportInfo) error {
+	for attempt := 0; attempt <= vmeInfo.DownloadRetries; attempt++ {
```

Comment:
nit: With this logic you actually try once more then what is specified. Like if you put there 4, it tries to download 5 times. Is this fine?

Reply:
My idea is that the retries count once you make the first attempt. For example, specifying 0 retries means attempting to download once and then doing 0 retries.